packages/font_iosevka/project.bri

```diff
@@ -0,0 +1,20 @@
+import * as std from "std";
+
+export const project = {
+  name: "iosevka",
```

Comment:
`project.name` should be similar to project directory name. We don't have yet a rule to check it with `brioche check`, but it's planned.

```suggestion
  name: "font_iosevka",
```